week-03/README.md

```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 3, Problem 1]
+- *Question* (sbagdi-stat6250): Which of the following is true for SAS statements?
+a) All SAS statements start and end with semi-colon.
+b) SAS statements cannot be combined in one line.
+c) SAS statements cannot continue over several lines.
+d) Usually begins with a SAS keyword and always end with a semi-colon.
+- *Answer* (sbagdi-stat6250): The correct answer is- Choice d.
 
 
 
 [Course Textbook Chapter 3, Problem 2]
-
+- *Question* (sbagdi-stat6250): What is the non-compulsory, but an essential good practice before resubmitting a revised SAS program?
 
 
 [Course Textbook Chapter 3, Problem 3]
-
+- *Question* (sbagdi-stat6250): After creating the abbreviations, what steps are to be performed to finally display the full length of the abbreviated text?
+- *Answer* (sbagdi-stat6250): Steps to display the full length abbreviated text-
+a) Type the abbreviation.
+b) if that abbreviation is recognized, a tooltip displays the expanded text; Press Tab Key or Enter to accept the abbreviation. 
 
  
 [Course Textbook Chapter 3, Problem 4]
-
+- *Question* (sbagdi-stat6250): What are the different types of syntax errors?
 
 
 [Course Textbook Chapter 3, Problem 5]
-
+- *Question* (sbagdi-stat6250): How to locate a statement that lacks a semi-colon in an error message?
 
 
 [Course Textbook Chapter 3, Problem 6]
-
+- *Question* (sbagdi-stat6250): Why doesn’t simply adding a quotation mark (in missing quotation mark errors) and resubmitting a program resolves the problem? 
 
 
 [Course Textbook Chapter 3, Problem 7]
-
+- *Question* (sbagdi-stat6250): What are the essential steps to resolve the issue of invalid option?
 
 
 [Course Textbook Chapter 3, Problem 10]
+- *Question* (sbagdi-stat6250): Which syntax error, other than the ‘missing run statement’, results in error message- ‘DATA Step running’ at the top of active window?
 
 
 
-[Course Textbook Chapter 4, Problem 1]
 
+[Course Textbook Chapter 4, Problem 1]
+- *Question* (sbagdi-stat6250): Which option is used to remove the default observation column (column that displays observation numbers)? Where do the programmer specifies that option in his program?
 
 
 [Course Textbook Chapter 4, Problem 3]
+- *Question* (sbagdi-stat6250): Why are logical operators used in any SAS programs? Also, Which operator can be used as an alternative to ‘OR’ operator? 
+- *Answer* (sbagdi-stat6250): The logical operators link sequence of expressions into compound expressions. Two types are: ‘AND’ (&) and ‘OR’ (|). Operator ‘IN’ can be used in place of ‘OR’ operator. 
 
 
 
 [Course Textbook Chapter 4, Problem 4]
-
+- *Question* (sbagdi-stat6250): What happens if a programmer doesn’t use OUT= option in a PROC SORT step? 
 
 
 [Course Textbook Chapter 4, Problem 7]
+- *Question* (sbagdi-stat6250): What does the DESCENDING option does? And to which statement should it be added?
```

Comment:
Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.